Applications/deviceconfiguration/faraday_config.ini

```diff
@@ -12,14 +12,14 @@ BOOT_FREQUENCY_MHZ = 914.5
 BOOT_RF_POWER = 20
 
 [GPS]
-DEFAULT_LATITUDE = 012345678
-DEFAULT_LATITUDE_DIRECTION = N
-DEFAULT_LONGITUDE = 0123456789
-DEFAULT_LONGITUDE_DIRECTION = W
-DEFAULT_ALTITUDE = 0
-DEFAULT_ALTITUDE_UNITS = M
+;NMEA DMM Format
+DEFAULT_LATITUDE_DIRECTION = N ; MAX LENGTH = 1 Byte
```

Comment:
While inline comments may have worked for you @kb1lqd they are not the standard from Windows which defined the INI file structure and therefore are not guaranteed to work.

https://stackoverflow.com/questions/1378219/do-standard-windows-ini-files-allow-comments#19550081